cost-analyzer/README.md

```diff
@@ -46,3 +46,4 @@ Parameter | Description | Default
 `serviceAccount.create` | Set this to `false` if you want to create the service account `kubecost-cost-analyzer` on your own | `true`
 `tolerations` | node taints to tolerate | `[]`
 `affinity` | pod affinity | `{}`
+`kubecostProductConfigs.productKey.mountPath` | Use instead of `kubecostProductConfigs.productKey.secretname` to declare the path at which the product key file is mounted (eg. by a secrets provisioner) | `N/A`
```

Comment:
Open question - I used `N/A` here given that this field is optional. If it is undefined, clients (eg. the `/getProductKey()` endpoint) will rely on reading at the fallback path `/var/configs/productkey/productkey.json`, but I thought that it wasn't necessary to share that information here

Reply:
I think that's fine.